docs/realtime/realtime.md

```diff
@@ -32,3 +32,10 @@ This section displays information about the current running connection such as t
 
 ## Programming interface
 Most of the time that you spend working with Realtime will be via its programming interface. More information is available on using this API in the [Realtime API Overview](./readme.md), and individual methods are documented on the [Realtime](../reference/classes/Normal.Realtime.Realtime.md) reference page.
+
+## Trouble connecting?
```

Comment:
I wonder if this would be better suited on the Common Questions page or the Getting Started page or both?

Reply:
Can we remove this section from the Realtime page if it's on Common Questions?